sdk/azcore/ci.yml

```diff
@@ -9,6 +9,6 @@ pr:
     - sdk/azcore/
     
 stages:
-- template: ../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+- template: ../eng/../pipelines/templates/jobs/archetype-sdk-client.yml
```

Comment:
Doesn't look right (applies to others).

Reply:
Not certain why this was fine for one of them, but not the rest. Pushed an update.